demos/remote-mcp-github-oauth/wrangler.jsonc

```
@@ -13,6 +13,9 @@
 			"tag": "v1"
 		}
 	],
+	"compatibility_flags": [
+		"nodejs_compat_v2"
```

```suggestion
		"nodejs_compat"
```

Shouldn't need v2 after 9/23/25